hail/python/hail/docs/change_log.md

```diff
@@ -22,8 +22,22 @@ Please note that **forward compatibility should not be expected, especially
 relating to file formats**: this means that it may not be possible to use
 an earlier version of Hail to read files written in a later version.
 
+---
+
+## Version 0.2.67
+
+### Critical bug fix
```

Comment:
This is a nit, but I might call this a "Critical Performance Fix" or something? I feel like language like "Critical Bug Fix" should maybe be reserved for us finding something that gives wrong answers? Feel free to disagree.

Reply:
good call, fixed